bin/atnodes

```diff
@@ -311,7 +334,9 @@ OPTIONS:
     -P            Prompt for passphrase (used for login,
                   could be privided by SSH_BATCH_PASSPHRASE).
     -tty          Pseudo-tty.
-    -q            Run SSH in quiet mode
+    -E            Use expect for input, such as password.
```

Comment:
Options documented in this usage message should be sorted alphabetically. `-E` shouldn't come after `-tty`.